google_api_lib/google_api_client.py

```diff
@@ -92,15 +92,15 @@ def add_puzzle_and_slack_links_to_sheet(self, puzzle_url, slack_channel_id,
                                             sheet_url):
         req_body = {
             'values': [
-                ['Puzzle link', puzzle_url],
-                ['Slack channel', '%s/app_redirect?channel=%s'
-                                  % (settings.SLACK_BASE_URL, slack_channel_id)]
+                [f'=HYPERLINK("{puzzle_url}", "Puzzle Link")'],
+                #['Slack channel', '%s/app_redirect?channel=%s'
+                #                  % (settings.SLACK_BASE_URL, slack_channel_id)]
```

Comment:
Why are these lines commented out? If we don't need them, we should probably just delete them.

Reply:
I was just being skittish about deleting code. I'll do it...

Reply:
haha same, but I guess we've got git to back us up? 💪 

anyway, thanks for the cleanup!